### ISSUE TITLE
Meet Ingress NGINX v1.2.0 and the chrooted NGINX process

### ISSUE THREAD

**Issue by megian** [projectsyn/component-ingress-nginx#36]
## Context
Check if the chrooted NGINX process should be supported by the component.

Reference:
* [Increasing the security bar in Ingress-NGINX v1.2.0](https://kubernetes.io/blog/2022/04/28/ingress-nginx-1-2-0/)
" [Changelog](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.2.0)

## Alternatives
Just keep it as is with less security

**Comment by glrf** [comment]
As far as I can tell, enabling this through the helm values just works. I'll probably just add a more convenient parameter to enable this.

The main question is, if we should enable it by default or not.

**Comment by glrf** [comment]
I tend to agree. There are just too many potential issues and we currently just don't have the means to test this on all the different environments this component currently runs on.

I'll just add a parameter to enable this and a short how-to explaining the impact.
